source/agora/utils/gc/GC.d

```diff
@@ -408,8 +408,9 @@ class ConservativeGC : GC
         return p;
     }
 
-
-    BlkInfo qalloc( size_t size, uint bits, const TypeInfo ti) nothrow
+    static if (__VERSION__ <= 2097)
+    {
+    override BlkInfo qalloc( size_t size, uint bits, const TypeInfo ti) nothrow
```

Comment:
Indentation missing?

Reply:
It was intentional, but I could also add it if you prefer.

Reply:
Was that because it is `static if`? Other places we still indent.